xml/art-obs-beginners-guide.xml

```diff
@@ -618,24 +621,32 @@ Cmnd_Alias  OSC_CMD = /usr/bin/osc, /usr/bin/build
      <filename>&obsworkdir1;/&gitproject;</filename>.
     </para>
     <para>
-     Download the sources from <link xlink:href="&gitupstream1;/releases"/>.
-     If the project has not published yet a release, download the latest sources
+     Download a TAR archive of the sources.
+    </para>
+    <para>
+     In our example, the project is hosted on &gh; and this URL can be used:
+     <link xlink:href="&gitupstream1;/releases"/>.
+     If there is no published release of a project yet, download the latest sources
      by clicking the <guimenu>Clone or download</guimenu> button and then the
      <guimenu>Download ZIP</guimenu> link.
     </para>
    </step>
    <step>
     <para>
-     Create a <firstterm baseform="Spec File">spec file</firstterm> which
-     contains metadata and build instructions. For RPM-based
-     distributions like &opensuse;, the basic structure should look like this:
+     Create the &packspec;. This file contains metadata and build
+     instructions.
+    </para>
+    <para>
+     As in this example we are building an RPM which can later be installed
+     on distributions like &opensuse;, create a <firstterm baseform="Spec
+     File">spec file</firstterm>. The basic structure should look like this:
     </para>
     <remark>toms 2017-08-09: Should the spec file be referenced by a GitHub
      link? How much detail do we need here?</remark>
```

Comment:
I think you miss the point here. 😉
This is not about how spec files are written in general. In this section, I describe a scenario with a specific spec file. That specific spec file is located on a GitHub project under an openSUSE domain (the "upstream project"). This makes it a bit more easier for the user to follow the details.

So the question is related to this *upstream project*: insert this relationship into the doc as a link or better describe it directly.